tools/cellxgene_census_builder/src/cellxgene_census_builder/release_manifest.py

```diff
@@ -18,6 +18,7 @@
     "CensusLocator",
     {
         "uri": str,  # resource URI
+        "relative_uri": str,  # relative URI
```

Comment:
good to synchronize these comments with the public API (e.g., which are obsolete, etc)

Reply:
Why do we even have two copies of those objects? Since the builder has the census as a dependency, maybe we could be more DRY here...

Reply:
True although as currently defined they are different (in absolute/relative URL fields).

But DRY is good, as long as we can keep our older package releases running.